### PR TITLE
[Hub Generated] Review request for Microsoft.Security to add version stable/2019-08-01

### PR DESCRIPTION
**Pull request by liranc**
This is a PR generated at [OpenAPI Hub](https://aka.ms/openapihub). You can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/liranc/azure-rest-api-specs/recommendations-2019-08-01...master/).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for AME onboarding.
- [x] [Validation tools](https://aka.ms/swaggertools) have been run on swagger spec(s) updated and errors have all been fixed with this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, [please view the FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist

- [x] Service team MUST add the "**WaitForARMFeedback**" label if the management plane API changes fall into one of the below categories.
*	adding/removing APIs.
* adding/removing properties.
* adding/removing API-version.
* adding a new service in Azure.

*Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.*

- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them.

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotAlertTypes.json [hunk 1]
```diff
@@ -208,6 +208,18 @@
           "example": "IoTSecurity",
           "description": "The name of the alert provider or internal partner"
         },
+        "productName": {
```
Comment (majastrz) [inline specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotAlertTypes.json:211]: >productName [](start = 9, length = 11)

As per [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md), evoluationary changes require a version bump.

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotRecommendations.json [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/iotSecuritySolutions/{solutionName}/iotRecommendations": {
```
Comment (majastrz) [inline specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotRecommendations.json:37]: >iotRecommendations [](start = 137, length = 18)

New APIs fall into a realm of evolutionary changes that require a version bump as per As per [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md), evoluationary changes require a version bump..

File: specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotRecommendations.json [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+          "type": "integer",
+          "description": "Total count of recommendations that conforms with the given filter options (not affected by page size)"
+        }
+      }
+    },
+    "IotRecommendation": {
+      "type": "object",
+      "description": "IoT recommendation",
+      "properties": {
+        "properties": {
```
Comment (pilor) [inline specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotRecommendations.json:184]: this is missing the top level resource properties "type", "name", and "id".  Please inherit from the common "Resource" definition like iotRecommendationTypes does

**Review by pilor — CHANGES_REQUESTED** [review]

**Comment by adrianhall** [comment]
Per email discussion with Azure API Review Board - Exception from policy is granted, and changes are APPROVED without a version bump.

In the future, additive / evolutionary changes will require a version bump.

**Review by pilor — APPROVED** [review]

**Comment by liranc** [comment]
@pilor @adrianhall Thanks for reviewing! Could you please merge the PR?

**Comment by pilor** [comment]
> @pilor @adrianhall Thanks for reviewing! Could you please merge the PR?

See step 8+ here:  https://armwiki.azurewebsites.net/rp_onboarding/process/api_review.html

**Comment by liranc** [comment]
> See step 8+ here: https://armwiki.azurewebsites.net/rp_onboarding/process/api_review.html

@pilor are there remaining comments/issues? As far as I understand steps 8+ are mostly about addressing feedback.

**Pull request merged**